doc/vagrant/share/packelektra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
It would be easy to expand the Vagrant-/Dockerfile to build environments that have Elektra installed.
If I add these two lines to the Dockerfile it generates an image with elektra 0.8.19:
```
ADD https://codeload.github.com/ElektraInitiative/libelektra/tar.gz/0.8.19 /tmp/elektra.tar.gz
RUN cd /tmp && tar xzf elektra.tar.gz && cd libelektra-0.8.19 \
  && mkdir build && cd build && cmake <your desired flags> .. && make -j && make install
```
I could add similar lines to the Vagrantfile.
(This approach has the drawback, that the resulting VM/image has unnecessary packages installed, that you only need to build Elektron but not to run it. As far as I know the best approach to tackle this is to build two containers: One that builds the package and another that just installs it).
I think if you provide such containers its best to distribute them via [dockerhub](https://hub.docker.com/) or [atlas](https://atlas.hashicorp.com/boxes/search) (but I think you have to pay for that).

I didn't do this because I had these two usecases in mind:

- Build your own customized package

- Try out a certain Elektra commit (useful if you want to review a pullrequest)